### PR TITLE
Sync MS Learn to FTK 11/12/2024

### PR DESCRIPTION
**Pull request by bandersmsft**
## 🛠️ Description

Fixes # 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [ ] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

File: docs-mslearn/toolkit/bicep-registry/scheduled-actions.md [hunk 1]
```diff
@@ -59,36 +57,40 @@ module <string> 'br/public:cost/<scope>-scheduled-action:1.1' = {
...
+| `scheduleWeeksOfMonth` | `array`  | Optional. List of weeks of the month that emails should be delivered. This value or scheduleDayOfMonth is required if scheduleFrequency is `Monthly`. Allowed: First, Second, Third, Fourth, Last. Default [] (not set).                                                                                    |
 | `scheduleStartDate`    | `string` | Optional. The first day the schedule should run. Use the time to indicate when you want to receive emails. Must be in the format yyyy-MM-ddTHH:miZ. Default = Now.                                                                                                                                    |
 | `scheduleEndDate`      | `string` | Optional. The last day the schedule should run. Must be in the format yyyy-MM-dd. Default = 1 year from start date.                                                                                                                                                                                   |
 
 <br>
 
 ## Examples
 
+The following examples  help you send an email on a schedule or when an anomaly is detected.
```
Comment (flanakin) [inline docs-mslearn/toolkit/bicep-registry/scheduled-actions.md:89]:
```suggestion
The following examples help you send an email on a schedule or when an anomaly is detected.
```

File: docs-mslearn/toolkit/help/contributors.md [hunk 2]
```diff
@@ -1,98 +1,83 @@
...
+ms.author: banders
+ms.date: 10/29/2024
+ms.topic: how-to
+ms.service: finops
+ms.reviewer: micflan
+#customer intent: As a FinOps user, I want to learn how to contribute to the FinOps toolkit by fixing bugs, suggesting new features, reviewing pull requests, and updating documentation.
 ---
 
-<span class="fs-9 d-block mb-4">Contribution guide</span>
+# FinOps contribution guide
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/contributors.md:13]:
```suggestion
# FinOps toolkit contribution guide
```

File: docs-mslearn/toolkit/help/contributors.md [hunk 3]
```diff
@@ -1,98 +1,83 @@
...
-- [Reviewing pull requests](#-reviewing-pull-requests)
-- [Answering questions](#-answering-questions)
-- [Updating documentation](#-updating-documentation)
-
-</details>
-
----
-
-Welcome to the FinOps toolkit! As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
+The FinOps Toolkit project welcomes contributions from the community. Welcome to the FinOps toolkit. As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective.
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/contributors.md:15]:
```suggestion
Welcome to the FinOps toolkit. As an open-source initiative, our success hinges on the collective effort of the community. Your contributions play a vital role in enhancing and expanding the toolkit's capabilities, making it more valuable and effective for everyone.
```

File: docs-mslearn/toolkit/help/contributors.md [hunk 4]
```diff
@@ -1,98 +1,83 @@
...
+- **Provide suggestions for new features or improvements:** If you're not comfortable with coding, you can still contribute by suggesting new features or improvements for our tools.
+- **Propose a new tool:** If you developed a FinOps-related tool and want it to be part of the FinOps toolkit, you can submit a proposal for consideration.
+- **Review pull requests:** Code reviews are a critical aspect of maintaining code quality and ensuring the reliability and functionality of the project. By reviewing existing pull requests (PRs) suggested by community members, you can contribute significantly to the project's success without necessarily creating new code.
+- **Answer questions:** Being actively involved in answering questions and providing assistance to community members is an invaluable contribution to the project.
+- **Update documentation:** Documentation serves as a crucial resource for users, developers, and contributors alike. We strive to keep documentation accurate, comprehensive, and up to date. It's essential for the success of the toolkit.
 
 We appreciate your interest in contributing to the FinOps Toolkit project!
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/contributors.md:26]:
```suggestion
We appreciate your interest in contributing to the FinOps toolkit!
```

File: docs-mslearn/toolkit/help/contributors.md [hunk 5]
```diff
@@ -101,23 +86,35 @@ Being actively involved in answering questions and providing assistance to commu
...
+## Related content
+
+Related products:
+
+- [Cost Management](/azure/cost-management-billing/costs/)
+
+Related solutions:
+
+- [FinOps toolkit Power BI reports](../power-bi/reports.md)
+- [FinOps hubs](../hubs/finops-hubs-overview.md)
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/contributors.md:119]:
```suggestion
```

File: docs-mslearn/toolkit/help/help-options.md [hunk 6]
```diff
@@ -30,3 +34,14 @@ Need help with an issue you're facing with one of the tools in the FinOps toolki
...
+## Related content
+
+Related products:
+
+- [Cost Management](/azure/cost-management-billing/costs/)
+
+Related solutions:
+
+- [FinOps toolkit Power BI reports](../power-bi/reports.md)
+- [FinOps hubs](../hubs/finops-hubs-overview.md)
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/help-options.md:47]:
```suggestion
```

File: docs-mslearn/toolkit/help/support.md [hunk 7]
```diff
@@ -44,6 +46,17 @@ If you run into an issue, we recommend taking the following actions:
...
+## Related content
+
+Related products:
+
+- [Cost Management](/azure/cost-management-billing/costs/)
+
+Related solutions:
+
+- [FinOps toolkit Power BI reports](../power-bi/reports.md)
+- [FinOps hubs](../hubs/finops-hubs-overview.md)
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/support.md:62]:
```suggestion
```

File: docs-mslearn/toolkit/help/troubleshooting.md [hunk 8]
```diff
@@ -165,3 +168,13 @@ If you're facing an error not listed above or need more help, file a [support re
 
 <br>
 -->
+## Related content
+
+Related products:
+
+- [Cost Management](/azure/cost-management-billing/costs/)
+
```
Comment (flanakin) [inline docs-mslearn/toolkit/help/troubleshooting.md:176]:
```suggestion
```

**Review by flanakin — APPROVED** [review]

**Review by Samdra31 — COMMENTED** [review]
[]()

**Pull request merged**